src/Sylius/Bundle/ResourceBundle/spec/Form/Type/ResourceChoiceTypeSpec.php

```diff
@@ -46,45 +55,43 @@ function it_should_be_a_form_type()
         $this->shouldImplement(FormTypeInterface::class);
     }
 
-    function it_has_a_parent_type_for_orm_driver()
+    function it_has_a_parent_type_for_orm_driver($metadata)
     {
-        $this->beConstructedWith('CountryModel', SyliusResourceBundle::DRIVER_DOCTRINE_ORM, 'sylius_country_choice');
+        $metadata->getDriver()->willReturn(SyliusResourceBundle::DRIVER_DOCTRINE_ORM);
+        $this->beConstructedWith($metadata);
```

Comment:
No need for that, if we use the same argument as in `let` it is auto mapped to the same object.